src/i18n/en.toml

```diff
@@ -40,6 +40,7 @@ to = "to"
 woman = "Woman"
 routeLengthUnit = "km"
 weightUnitGram = "g"
+carbonDioxideWeightUnit = "g CO2"
```

Comment:
CO2 should be HTML formatted text CO<sup>2</sup> 

```html
CO<sup>2</sup> 
```

Reply:
Yes, good point. In this case it's <sub>, but you're right.